.github/workflows/parachain.yml

```diff
@@ -103,6 +104,17 @@ jobs:
           profile: minimal
           toolchain: 1.62.1
           target: wasm32-unknown-unknown
+      - name: test beacon client with minimal feature
+        uses: actions-rs/cargo@v1
+        with:
+          command: test
+          args: >-
+            --manifest-path parachain/Cargo.toml
+            --verbose
+            --package snowbridge-ethereum-beacon-client
+            --features runtime-benchmarks
+            --features minimal
```

Comment:
Why do we need to add this?

Reply:
to cover the minimal feature which is not included in current CI